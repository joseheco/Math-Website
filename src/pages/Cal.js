import React from 'react';
import Calculator from '../components/Calculator';

const RenderCal = () => (
  <div className="container">
    <p className="cal-p"> Lets do some math!</p>
    <Calculator />
  </div>
);

export default RenderCal;
