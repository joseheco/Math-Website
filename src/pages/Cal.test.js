import { render } from '@testing-library/react';
import React from 'react';
import Cal from './Cal';

test('Calculator page renders', () => {
  const tree = render(<Cal />);
  expect(tree.toJSON).toMatchSnapshot();
});
