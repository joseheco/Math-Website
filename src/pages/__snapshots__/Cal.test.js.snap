// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator page renders 1`] = `undefined`;
